thinkhazard/static/js/report.js

```diff
@@ -210,7 +97,40 @@
   function showOverview() {
     $('ul.hazards a').filter('[href="#overview"]').tab('show');
     hazardType = undefined;
-    addMap(hazardType);
+    showHazard();
     $('#legend').hide();
   }
+
+  /**
+   * Adds or replaces the hazard image layer.
+   */
+  function showHazard(hazardType) {
+    if (hazardLayer) {
+      map.removeLayer(hazardLayer);
```

Comment:
There's a `setSource` method that we could use instead of re-creating the layer.